test/specs/WindowSizeSpec.ts

```diff
@@ -0,0 +1,58 @@
+import { assert } from "chai";
+import { BrowserUtils } from "../../src/commons/BrowserUtils";
+import Size = WebdriverIO.Size;
+import { Reporter } from "../../src/commons/Reporter";
+
+// tslint:disable-next-line:no-http-string
+const sampleAppUrl: string = 'http://127.0.0.1:8000/';
+
+/**
+ * wdio-allure-ts Change window size and get window size (setWindowSize, getWindowSize)
+ */
+describe('WindowSizeSpec of BrowserUtils Tests', () => {
+    it('Validate get window size default', () => {
+        BrowserUtils.navigateToUrl(sampleAppUrl);
+        const size : Size = BrowserUtils.getWindowSize();
```

Comment:
window.innerWidth
window.innerHeight
are JS call for getting the dimensions for comparison

Reply:
Runing script doesn't work as expected.

Reply:
done